api/tests/opentrons/protocol_engine/commands/heater_shaker/test_open_labware_latch.py

```diff
@@ -43,5 +43,6 @@ async def test_open_labware_latch(
     ).then_return(hs_hardware)
 
     result = await subject.execute(data)
+    decoy.verify(hs_module_substate.raise_if_shaking(), times=1)
     decoy.verify(await hs_hardware.open_labware_latch(), times=1)
```

Comment:
I think it's important to verify call order here:
```suggestion
    decoy.verify(
        hs_module_substate.raise_if_shaking(),
        await hs_hardware.open_labware_latch(),
    )
```

Reply:
Ah, I naively thought that `times=1` is functionally equivalent to not specifying `times` and does call order verification. Now I see why it wouldn't do that